olm/operator-resources/instana-agent-operator.yaml

```diff
@@ -79,6 +79,7 @@ rules:
   - 'update'
   - 'list'
   - 'watch'
+  - 'create'
```

Comment:
I think this may cause problems if the new operator image gets re-pulled by existing operator installations and requires an update.

Would be nice if the error message that we show in the operator if we fail on an API call said something like "Please check that your operator has the cluster role permissions that match this operator version".

Actually, ideally we would be a bit more careful with our versioning e.g. we would hard code the major version of the operator in the yaml and push major version tags as well as `latest`, so that we can avoid re-pulling the image causing permissions issues.

Reply:
Added some changes to address this @wiggzz. Please re-review ♻️ when you get a chance 🙏 ❤️ !